Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8-jdk-alpine AS builder
```

Comment:
Can this be made to use java11? I read that graalvm supports java11 now, and I'm about to use it in another PR

Reply:
maybe @dsyer can chime in

Reply:
I haven't tested it. It works with the GraalVM version proposed, which is Java 8. If I were you I would wait a bit before you push hard on Java 11 (technically now supported for native images but only just).

Reply:
Hmm, did this run fine locally?

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fbe535c0216, pid=27, tid=50
#
# JRE version: OpenJDK Runtime Environment (11.0.5+10) (build 11.0.5+10-jvmci-19.3-b05-LTS)
# Java VM: OpenJDK 64-Bit GraalVM CE 19.3.0 (11.0.5+10-jvmci-19.3-b05-LTS, mixed mode, sharing, tiered, jvmci, jvmci compiler, compressed oops, g1 gc, linux-amd64)
```
from https://github.com/projectriff/streaming-processor/runs/324809611